README.md

```diff
@@ -1,9 +1,8 @@
-<a href="https://pytorch.org/get-started/locally/"><img alt="PyTorch" src="https://img.shields.io/badge/PyTorch-ee4c2c?logo=pytorch&logoColor=white"></a>  [![report](https://img.shields.io/badge/ArXiv-Paper-blue)](https://arxiv.org/abs/2402.09611)
-
+<a href="https://pytorch.org/get-started/locally/"><img alt="PyTorch" src="https://img.shields.io/badge/PyTorch-ee4c2c?logo=pytorch&logoColor=white"></a> [![report](https://img.shields.io/badge/ArXiv-Paper-blue)](https://arxiv.org/abs/2402.09611)
```

Comment:
Can you revert the changes you made to this README?

Reply:
done.